SSDynamicText/NSAttributedString+SSTextSize.h

```diff
@@ -3,10 +3,10 @@
 //  SSDynamicText
 //
 //  Created by Remigiusz Herba on 28/08/15.
-//
+//  Copyright (c) 2015 Splinesoft. All rights reserved. 
 //
 
-#import <Foundation/Foundation.h>
+@import Foundation;
```

Comment:
CocoaPods' generated library doesn't have the module flag enabled, so adding this flag in the podspec ensures that modules are enabled for the library. I haven't checked in a while; perhaps CocoaPods has changed or improved here?